__tests__/app.test.js

```diff
@@ -331,3 +331,54 @@ describe.only("GET /api/users", () => {
       });
   });
 });
+
+describe.only("GET /api/articles (filter)", () => {
+  test("200: Returns an array of articles, sorted by created_at in descending order by default", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toBeInstanceOf(Array);
```

Comment:
This assertion becomes redundant as we use a forEach in this test - if it wasn't an array we wouldn't be able to and the test would fail so we don't explicitly need to check that the articles are an array :slightly_smiling_face:

Reply:
I was looking through the Jest documentation to learn about some new methods, and I didn't even realise that I had already done so. 😊